### PR TITLE
Chore: release 4.0.0

### PR DESCRIPTION
**Pull request by moitias**
- update README and CHANGELOG for 4.0.0
- waiting for #90 to be merged first

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,31 @@
 All notable changes to this project will be documented in this file.
 See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)
 
-## 3.0.0 (Unreleased)
+## 4.0.0
+
+### Added
+
+- add support for Object Storage
+- support io.Reader as SourceLocation in CreateStorageImport
```
Comment (ajmyyra) [inline CHANGELOG.md:11]: This is just :100:

**Review by ajmyyra — APPROVED** [review]
All good!

**Comment by Darep** [comment]
Should we bump the version in internal/globals.go in this PR?

**Comment by moitias** [comment]
> Should we bump the version in internal/globals.go in this PR?

Great catch, will fix!

**Pull request merged**